Makefile

```diff
@@ -44,4 +44,11 @@ build-gadget-container:
 
 .PHONY: test
 test:
-	go test ./...
+	go test -test.v ./...
+
+.PHONY: integration-tests
+integration-tests:
+	KUBECTL_GADGET="$(shell pwd)/kubectl-gadget-linux-amd64" \
+		go test -test.v ./integration/... \
```

Comment:
What about omitting `-test.v` to avoid that noise in the output?

Reply:
ok. Initially, I added it to have all the logs in the GitHub Actions logs for debugging. But if the test fails, I think it will print the details anyway.

Reply:
Yes, it's printed if the test fails and it's a nice behavior because avoid polluting the output with information about passed tests.